src/app/scripts/cac/user/cac-user-preferences.js

```diff
@@ -76,6 +84,17 @@ CAC.User.Preferences = (function(Storages, _) {
     }
 
     /**
+     * Check if value has been set by user, or is a default.
+     * Will return false if given preference does not exist.
+     *
+     * @param {String} preference Name of setting to check
+     * @return {Boolean} True if getPreference will return a default value
+     */
+    function isDefault(preference) {
+        return !_.has(options, preference) && _.has(defaults, preference);
```

Comment:
Nice and tidy.

It's making me wonder, though, about what the right behavior is when someone sets an option to the default value.  This will be false.  Is that what we want?  If not we could address that either here or in `storage.set()`, by checking incoming values against the default and deleting them if they match.

But maybe it is intended.  If the idea is for people to see their selections in the modal, it might make sense to show them even if they pick default values.